File: rust/src/models/onboarding_mode.rs
// this file is @generated
use std::fmt;

use serde::{Deserialize, Serialize};

/// Onboarding mode for connected accounts
#[derive(
    Clone, Copy, Debug, Default, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize,
)]
pub enum OnboardingMode {
    #[default]
    #[serde(rename = "express")]
    Express,

    #[serde(rename = "full")]
    Full,
}

impl fmt::Display for OnboardingMode {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        let value = match self {
            Self::Express => "express",
            Self::Full => "full",
        };
        f.write_str(value)
    }
}

File: typescript/src/models/onboardingMode.ts
// this file is @generated
/** Onboarding mode for connected accounts */
export enum OnboardingMode {
  Express = "express",
  Full = "full",
}

export const OnboardingModeSerializer = {
  _fromJsonObject(object: any): OnboardingMode {
    return object;
  },

  _toJsonObject(self: OnboardingMode): any {
    return self;
  },
};
